discobot.go

```diff
@@ -87,7 +89,9 @@ func (bot *DiscoBot) playSound(s *discordgo.Session, guildID, channelID, url str
 		if packet.Timecode == webm.BadTC {
 			r.Shutdown()
 		} else {
-			vc.OpusSend <- packet.Data
+			if bot.Playstatus {
+				vc.OpusSend <- packet.Data
```

Comment:
In this case, stream chunks will be skipped. You have to wait here until the user will call `play` command.
1. Create `startPlayback chan struct{}` in `DiscoBot` struct
2. In the case when `PlayStatus=true` try to read from `startPlayback` channel.
3. In the `play` command handler write to `startPlayback` channel to continue playing.

Reply:
**done**